sdk/cognitivelanguage/Azure.AI.Language.QuestionAnswering/src/Generated/QuestionAnsweringProjectsClient.cs

```diff
@@ -1262,7 +1262,7 @@ public virtual Response GetUpdateQnasStatus(string projectName, string jobId, Re
             }
         }
 
-        /// <summary> Add Active Learning feedback. </summary>
+        /// <summary> Update Active Learning feedback. </summary>
```

Comment:
That's a good point, I'm not too sure. Let me check with Rohan.

Reply:
I've checked with Rohan and "Add" seems like the more appropriate word. It seems like this was a change made in [this swagger PR](https://github.com/Azure/azure-rest-api-specs/pull/17082), but it ended up being closed and not included in the final swagger changes.

I'll edit it with a swagger transform for now (since I don't think a change to the swagger would be merged in time for code complete).

Reply:
I've checked with Rohan and "Add" seems like the more appropriate word. It seems like this was a change made in [this swagger PR](https://github.com/Azure/azure-rest-api-specs/pull/17082), but it ended up being closed and not included in the final swagger changes.

I'll edit it with a swagger transform for now (since I don't think a change to the swagger would be merged in time for code complete).